tardis/montecarlo/base.py

```diff
@@ -686,8 +665,17 @@ def from_config(
             config.montecarlo.tracking.initial_array_length
         )
 
+        if packet_source is None:
```

Comment:
@andrewfullard Please check this regarding [your previous comment](https://github.com/tardis-sn/tardis/pull/2329/files/7babf61a6a006a99be77d8bd0958ee593e099a54#r1232335361).